vars/piperPipelineStageInit.groovy

```diff
@@ -43,7 +43,12 @@ import static com.sap.piper.Prerequisites.checkScript
      */
     'projectName',
     /**
-     * Defines the library resource containing stage/step initialization settings. Those define conditions when certain steps/stages will be activated. **Caution: changing the default will break the standard behavior of the pipeline - thus only relevant when including `Init` stage into custom pipelines!**
+     * Specify to execute artifact versioning in a kubernetes pod.
+     * @possibleValues `true` , `false`
+     */
+    'runOnPod',
```

Comment:
The parameter name does not tell me anything. What should run in(?) the pod? How about adding the versioning to the parameter name, as for example in:
`runVersioningInPod`